libs/components/atoms/src/lib/label-box/label-box.component.ts

```diff
@@ -0,0 +1,15 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+/** Label box component for reuse accross the page. */
```

Comment:
Add to documentation that content with the `end` class will be put at the end of the label box